### PR TITLE
✨ Issue#27 create reusable content text component

### PR DESCRIPTION
**Pull request by jbdura**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - Added reusable 

## Type of change
Added reusable component

Please delete options that are not relevant.

## Screenshot (optional)

Screenshot from 2023-08-11 13-47-10

## Checklist

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings

### CODE REVIEW

**Comment by jbdura** [comment]
Hello, done the required changes

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
You've mixed two issues into one. Kindly sort that.

File: libs/elements/layout/header/src/lib/header/header.component.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  selector: 'elewa-website-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+})
+export class HeaderComponent {
+  /** 
+  * This is a header component class. 
+  * With this class, you can link the header section
+  */
+
```
Comment (LemmyMwaura) [inline libs/elements/layout/header/src/lib/header/header.component.ts:13]: No need for this. Also, JSDoc comments go above the item your describing.

File: libs/elements/layout/header/src/lib/header/header.component.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  // To check if navigation menu is open or not
+  isMenuOpen = false;
+  toggleMenu() {
+    this.isMenuOpen = !this.isMenuOpen;
+  }
+
+  constructor () {
+    
+  }
```
Comment (LemmyMwaura) [inline libs/elements/layout/header/src/lib/header/header.component.ts:25]: remove the empty constructor

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
You've mixed two different issues. Kindly fix that.

**Pull request closed without merging**